gtests/net/mptcp/common/multi-ep.sh

```diff
@@ -65,7 +65,7 @@ else # ipv4 or ipv4-mapped-ipv6
     if [[ $OPT_LOCAL_IP =~ ([0-9]+[.][0-9]+[.][0-9]+[.])([0-9]+) ]]; then
         network=0
         IFS=. read a b c d <<-EOF
-       `echo $OPT_LOCAL_NETMASK`
+       `echo $OPT_NETMASK_IP`
 EOF
```

Comment:
detail: maybe clearer with:

```bash
IFS=. read a b c d <<< "${OPT_NETMASK_IP}"
```

But up to you ;-)

Reply:
yes but that triggers this:
```
 SC3011: In POSIX sh, here-strings are undefined.
```
(it's true that packetdrill scripts are not expected to pass the shelllcheck, though. But if there is the possibility, I generally prefer the POSIX version)

Reply:
That's strange you have this warning because the shebang is pointing us to Bash, not `/bin/sh`.
Or do we need to change the shebang too?

(there are plenty other warnings from shellcheck in the script :-D we should use `read -r`, `$(...)` instead of backquotes, etc)

I mainly reacted because it was strange to see we evaluate `echo` instead of using the variable directly:

```bash
IFS=. read -r a b c d <<-EOF
${OPT_NETMASK_IP}
EOF
```

But as long as it works, up to you ;-)